ksql-execution/src/test/java/io/confluent/ksql/execution/builder/KsqlQueryBuilderTest.java

```diff
@@ -260,7 +261,14 @@ public void shouldTrackSchemasTakingIntoAccountSerdeOptions() {
     );
 
     // Then:
-    assertThat(ksqlQueryBuilder.getSchemas().toString(),
-        is("fred.context = BOOLEAN"));
+    assertThat(formatSchemas(), is("fred.context = BOOLEAN"));
+  }
+
+  private String formatSchemas() {
```

Comment:
don't think this change is needed if you move QuerySchemas into common, as suggested above.